Initial draft of the method section (#9)
Ctrl-Alt-Defeat-Foundation-Models/informatiCup2024Paper: paper.tex

\subsubsection{Text detector models}

\section{Method}

> To test and evaluate the effect of different data augmentation techniques on the detection capabilities of fake image or text detectors, we created a pipeline of three components: The first component is a generator, which is a model capable of generating fake images or texts. The second component is a processor which augments the data through a specified method, after inputting a generated image or text. The last component is a detector, which suggests an input being either real or fake. This way we can compare it to the ground truth, to evaluate the capabilities of the detector. By comparing the results with augmented artefacts and with the generated ones without augmentation, we can evaluate the effectiveness of different AI detector-tricking techniques.

Reviewer: ```suggestion
To test and evaluate the effect of different data augmentation techniques on the detection capabilities of fake image or text detectors, we created a pipeline of three components: The first component is a generator, which is a model capable of generating fake images or texts. The second component is a processor augmenting the given image or text through a specified method. The last component is a detector, which suggests an input being either real or fake. This way we can compare it to the ground truth, to evaluate the capabilities of the detector. By comparing the results with augmented artefacts and with the generated ones without augmentation, we can evaluate the effectiveness of different AI detector-tricking techniques.
```

Reviewer 2: ```suggestion
To test and evaluate the effect of different data augmentation techniques on the detection capabilities of fake image or text detectors, we created a pipeline of three components: The first component is a generator, which is a model capable of generating fake images or texts. The second component is a processor which augments the data through a specified method, after given a generated image or text. The last component is a detector, which suggests an input being either real or fake. This way we can compare it to the ground truth, to evaluate the capabilities of the detector. By comparing the results with augmented artefacts and with the generated ones without augmentation, we can evaluate the effectiveness of different AI detector-tricking techniques.
```

> To test and evaluate the effect of different data augmentation techniques on the detection capabilities of fake image or text detectors, we created a pipeline of three components: The first component is a generator, which is a model that is capable of generating fake images or text. The second component is a processor. After inputting a generated image or text, this component augments the data with a specified method. The third component contains a detector, which returns whether an input is a fake artefact or a real one. This way we can compare it to the ground truth, to evaluate the capabilities of the detector. By comparing the results with augmented artefacts and with the genrated artefacts without augmentation, we can see how effective different techniques are to trick ai detectors.

Reviewer 2: ```suggestion
To test and evaluate the effect of different data augmentation techniques on the detection capabilities of fake image or text detectors, we created a pipeline of three components: The first component is a generator, which is a model capable of generating fake images or texts. The second component is a processor which augments the data a specified method, after inputting a generated image or text. The last component contains a detector, which returns whether the input is a fake artefact or a real one. This way we can compare it to the ground truth, to evaluate the capabilities of the detector. By comparing the results with augmented artefacts and the generated ones without augmentation, we can see how effective different techniques are to trick ai detectors.
```

> To test and evaluate the effect of different data augmentation techniques on the detection capabilities of fake image or text detectors, we created a pipeline of three components: The first component is a generator, which is a model that is capable of generating fake images or text. The second component is a processor. After inputting a generated image or text, this component augments the data with a specified method. The third component contains a detector, which returns whether an input is a fake artefact or a real one. This way we can compare it to the ground truth, to evaluate the capabilities of the detector. By comparing the results with augmented artefacts and with the genrated artefacts without augmentation, we can see how effective different techniques are to trick ai detectors.

Reviewer: Would do it like this:
```suggestion
To test and evaluate the effect of different data augmentation techniques on the detection capabilities of fake image or text detectors, we created a pipeline of three components: The first component is a generator, which is a model capable of generating fake images or texts. The second component is a processor agmenting the input data, which would can be an image or an text, through a specified method. The last component is a detector, which suggests an input being either real or fake. This way we can compare it to the ground truth, to evaluate the capabilities of the detector. By comparing the results with augmented artefacts and the generated ones without augmentation, we can evaluate the effectiveness of different AI detector-tricking techniques.
```
